ACE/ace/OS_NS_unistd.inl

```diff
@@ -929,17 +954,27 @@ ACE_OS::sleep (const ACE_Time_Value &tv)
   //FUZZ: enable check_for_lack_ACE_OS
 #else
 # if defined (ACE_HAS_NONCONST_SELECT_TIMEVAL)
+#   if !defined (ACE_LACKS_SELECT)
```

Comment:
Maybe move ACE_LACKS_SELECT before the check for ACE_HAS_NONCONST_SELECT_TIMEVAL, would only be one time necessary at that moment instead of 2 times here